packages/commonwealth/client/scripts/views/modals/delete_address_modal.tsx

```diff
@@ -57,9 +52,12 @@ export const DeleteAddressModal = ({
       if (response?.data.status === 'Success') {
         const updatedAddresses = [...user.addresses].filter(
           (a) =>
-            a.addressId !== address.addressId &&
+            a.addressId !== address.addressId ||
```

Comment:
this should probably be `&&` instead of `||` since a user can have a single address connected to multiple communities.